src/SDL_log.c

```diff
@@ -767,10 +786,15 @@ static void SDLCALL SDL_LogOutput(void *userdata, int category, SDL_LogPriority
             (void)fclose(pFile);
         }
     }
+#elif defined(SDL_PLATFORM_NGAGE)
+    {
+        /* Nothing to do here. */
+    }
 #endif
 #if defined(HAVE_STDIO_H) && \
+    !(defined (SDL_PLATFORM_NGAGE) || \
```

Comment:
extra parentheses look real bad here. You don't want the fprintf() wit ngage, yes? Why not just insert `!defined (SDL_PLATFORM_NGAGE) && \ ` in there as a one-liner?

Reply:
It's pretty similar to L796 and L797. I'm not sure what would be better here.